docs/chapters/05_going-deeper.md

```diff
@@ -166,6 +166,92 @@ source <path-to-your-bash-file> && boost deploy -e production
 
 Now just let the magic happen, Booster will create everything for you and give you back your app ready to use URL. 🚀
 
+#### Azure synth command
+
+Azure provider implement the experimental **Booster** `synth` command. This command will generate [Terrafom](https://www.terraform.io/) templates from your code. It will also generate needed files to deploy your Booster application using [cdktf](https://learn.hashicorp.com/tutorials/terraform/cdktf).
+
+Running `synth` command, for example `boost synth -e production` will generate following files:
+
+* A file `cdktf.json`: A basic json file to deploy your application using `cdktf`
+* A folder `cdktf.out`: with the Terraform templates.
+
+Booster deploy command for Azure will deploy your application using the generated templates.  You don't need to run the `synth` command for deploy your application, the `deploy` command will generate the templates before deploy for you.
```

Comment:
One question here, if you've generated the templates and you run `boost deploy` will Booster generate and overwrite the templates or will it use the existing ones?

Reply:
It will overwrite the templates and generate a new zip file with the functions